libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -254,6 +258,11 @@ - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
 - (void)layoutWebView {
```

Comment:
Changed because this is what it looked like before
<img width="901" alt="nav bar webview" src="https://user-images.githubusercontent.com/840431/114227803-52b1eb00-992a-11eb-89d0-98c8e90db678.png">

iPhone/iPad still look normal with the change
<img width="1409" alt="iphone-ipad" src="https://user-images.githubusercontent.com/840431/114228089-bf2cea00-992a-11eb-9ce3-39b0b8c02ce0.png">